hedera-mirror-rosetta/app/middleware/health.go

```diff
@@ -0,0 +1,86 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+package middleware
+
+import (
+	"fmt"
+	"github.com/coinbase/rosetta-sdk-go/server"
+	"github.com/hashgraph/hedera-mirror-node/hedera-mirror-rosetta/types"
+	"github.com/hellofresh/health-go/v4"
+	"github.com/hellofresh/health-go/v4/checks/postgres"
+	"time"
+)
+
+const (
+	livenessPath  = "/health/liveness"
+	readinessPath = "/health/readiness"
+)
+
+// HealthController holds data used to response to health probes
+type HealthController struct {
```

Comment:
```suggestion
type healthController struct {
```

don't have to export the struct

Reply:
done